cmd/blockchaincmd/add_validator.go

```diff
@@ -124,6 +126,8 @@ Testnet or Mainnet.`,
 	cmd.Flags().StringVar(&nodeEndpoint, "node-endpoint", "", "gather node id/bls from publicly available avalanchego apis on the given endpoint")
 	cmd.Flags().Uint64Var(&weight, validatorWeightFlag, uint64(constants.DefaultStakeWeight), "set the weight of the validator")
 	cmd.Flags().StringVar(&validatorManagerOwner, "validator-manager-owner", "", "force using this address to issue transactions to the validator manager")
+	stakerPrivateKeyFlags.SetFlagNames("staker-private-key", "staker-key", "staker-genesis-key")
+	stakerPrivateKeyFlags.AddToCmd(cmd, "as source of staking funds")
```

Comment:
how is the provider of this fund different from the provider of the fund for previous steps (provider of initial balance to validators, PoA)